Portfolio/src/Wrapper.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <div class="styled-wrapper">
+
+    <slot></slot>
+  </div>
+</template>
+
+<style scoped>
+.styled-wrapper {
+  font-family: "Raleway";
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 2rem;
+}
+
+@media (min-width: 280px) and (max-width: 767px) {
+  .styled-wrapper {
+    align-items: center;
+    margin: 5rem;
+  }
+}
+
+@media (min-width: 1900px) {
+  .styled-wrapper {
+    margin: 3rem 8rem 0 8rem;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1024px) {}
```

Comment:
I would recommend removing all redundant code and be strict about that from the get-go. It will ease the clean up later on (which usually never happens because we developers are lazy and there are always changing priorities).

Reply:
Pair coded with @Evgenios95